src/Entities/SplashPotionEntity.cpp

```diff
@@ -57,26 +57,21 @@ void cSplashPotionEntity::OnHitEntity(cEntity & a_EntityHit, Vector3d a_HitPos)
 
 void cSplashPotionEntity::Splash(Vector3d a_HitPos)
 {
-	m_World->ForEachEntity([=](cEntity & a_Entity)
+	double XZCalculation = 8.25/2;
+	double YCalculation = 4.25/2;
```

Comment:
Radius, and Height would be more intuitive, especially when using the simpler constructor below.

Reply:
Actually now that I think about it, the wiki mentions is a cuboid calculation and not a sphere. Unless I'm mistaken, wouldn't a Radius object (and the cBoundingBox constructor) be incorrect?

Reply:
Maybe it's not more inuitive then! It's just a different name for the same thing -> Radius == X/Z Distance from the centre, Height == Y Distance above the given point

https://github.com/cuberite/cuberite/blob/89fb16fc2717f24be516d2425e415b16120d07b4/src/BoundingBox.cpp#L32

Either way, the way you've done it already is fine, no need to make a change. I was just pointing out the alternative function available.

Reply:
Ohhh Yes I see. Well intuitive may have nothing to do with it :) I'm a business programmer by day, so game programming is a bit of a challenge for me (in a good way!)